meta/collection-requirements.yml

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: MIT
+collections:
+  - community.windows
+  - ansible.windows
```

Comment:
Are these needed at runtime, or only at test-time?

Reply:
These are only for test-time, I have added it according to advice from spetrosi in irc #system-roles channel.

Reply:
`meta/collection-requirements.yml` is only for runtime requirements - I'm working on a patch which will allow us to use `tests/collection-requirements.yml` for test-time requirements.

Reply:
Ok, moved it.